docs/usage/docker.md

```diff
@@ -206,6 +206,41 @@ module.exports = {
 };
 ```
 
+#### Google Container Registry
+
+Assume you are running GitLab CI in the Google Cloud, and you are storing your Docker images in the Google Container Registry (GCR).
+
+Access to the GCR uses Bearer token based authentication.
+This token can be obtained by running `gcloud auth print-access-token`, which requires the Google Cloud SDK to be installed.
+
+Storing the bearer token for subsequent builds in a variable (like you can do with the `RENOVATE_TOKEN`) is not an option because the token expires after 60 minutes.
+
+When running Renovate in this context the Google access token must be retrieved and injected into the `hostRules` configuration just before Renovate is started.
+
+_This documentation gives **a few hints** on **a possible way** to achieve this end result._
+
+The basic approach documented here is that you create a custom image and then run Renovate as one of the stages of your project.
+To make this run independent of any user you should use a `Project Access Token` for the project and use this as the `RENOVATE_TOKEN` variable for Gitlab CI.
```

Comment:
![image](https://user-images.githubusercontent.com/240096/122213845-b10ac500-cea9-11eb-8bbe-c7ed18e06fdd.png)

Reply:
Thank you! Just checking. 😄

Reply:
Do note that GitLab also has Personal Access Tokens! It has both.
I think that it is better to tie something like this to the project instead of the person who happens to be in the team.

Reply:
![image](https://user-images.githubusercontent.com/240096/122220778-f383d000-ceb0-11eb-95c9-8a3f9c954ae7.png)